test/foundry/utils/Constants.sol

```diff
@@ -67,4 +67,37 @@ library Constants {
   function getDropsEventConfig() internal pure returns (uint256) {
     return 0x0000000000000000000000000000000000000000000000000000000000040000;
   }
+
```

Comment:
Seeing that this is growing, I think it would be a good idea to create a new directory under `foundry` called `constants` and then create a new file for contract specific constants. We can decide on a naming convention, but maybe something like `CustomERC721.Constants.sol`. `Core.Constants.sol`, etc